dhcpv6/ztpv6/parse_vendor_options.go

```diff
@@ -92,3 +100,32 @@ func ParseVendorData(packet dhcpv6.DHCPv6) (*VendorData, error) {
 	}
 	return nil, errors.New("failed to parse vendor option data")
 }
+
+type MlnxSubOption int
```

Comment:
Maybe use uint16 for consistency with:

https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/types.go#L78